### ISSUE TITLE
Fibrofatty replacement of right ventricular myocardium 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7596]
**Preferred term label:**
Fibrofatty replacement of right ventricular myocardium 

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://pubmed.ncbi.nlm.nih.gov/16549640/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
